static/js/util/util.js

```diff
@@ -256,6 +258,21 @@ export function getLocation(profile: Profile): string {
   return `${subCountryLocation}, ${countryLocation}`;
 }
 
+/**
+ * returns the user's most recent (or current) employer
+*/
+export function getEmployer(profile: Profile): Maybe<string> {
+  let entries = workEntriesByDate(profile.work_history);
+  if ( _.isEmpty(entries) ) {
+    return Nothing();
+  }
+  let [, entry] = entries[0];
+  if ( entry.company_name ) {
+    return Just(entry.company_name);
+  }
+  return Nothing();
+}
+
```

Comment:
general comment, not something that's needed for this PR: from what i can see, `static/js/util/util.js` doesn't seem like it exists to provide utility functions. it seems like the place where we just dump any functionality that we don't have a logical place for. there are a bunch of functions in here that we use to extract/calculate information about a Profile (add this one to the list). at the very least, i feel like we should create a `static/js/util/profile.js` file and move those functions into there. i would say that's still a little iffy since it's still in a 'util/' folder and it isn't really 'utility' functionality, but probably a step in the right direction. what do you think?